docs/dev/libvirt/README.md

```diff
@@ -12,7 +12,7 @@ Before you begin, install the [build dependencies](../dependencies.md).
 Make sure you have KVM enabled by checking for the device:
 
```

Comment:
" Fixes #708 " is not a good commit summary. The shortlog is for summarising the change. Please swap the commit details with the shortlog and you're golden. :)

Reply:
Done, thanks! :)